src/composables/useNetwork.ts

```diff
@@ -0,0 +1,31 @@
+import { ref } from 'vue';
+
+export type NetworkId = 1 | 3 | 4 | 5 | 42 | 137 | 42161;
+
+export enum Network {
+  MAINNET = 1,
+  ROPSTEN = 3,
+  RINKEBY = 4,
+  GÖRLI = 5,
+  KOVAN = 42,
+  POLYGON = 137,
+  ARBITRUM = 42161
+}
+
+const DEFAULT_NETWORK =
+  process.env.VUE_APP_NETWORK != null
+    ? (Number(process.env.VUE_APP_NETWORK) as NetworkId)
+    : Network.MAINNET;
+
+export const networkId = ref<NetworkId>(DEFAULT_NETWORK);
```

Comment:
Changing a network is as easy as changing this initial value to something else. Try, refresh and see!